doc/source/examples/rpt/photon-count-calculation-in-a-cylindrical-vessel/photon-count-calculation-in-a-cylindrical-vessel.rst

```diff
@@ -0,0 +1,325 @@
+
+The last case studied (Case IV) shows the evolution of the photon count when :math:`\mu_d` is so great that :math:`f_d(\alpha_j, \theta_j)` tends to :math:`1 \ \forall y \in ]-10, 10[` cm. By doing so, we can see the evolution of the count when the efficiency is independent of the interaction between the emitted :math:`\gamma`-ray and the detector. With this case, we isolate the effect of the evolution of :math:`f_a(\alpha, \theta)` on the count. More specifically, we're looking at the evolution of :math:`e(\alpha,\theta)` as the particle travels in the vessel, since :math:`\mu_r` remains constant in the studied domain. We notice that we have a local minimum at :math:`y \approx -4.6` where we saw the convex section on the Case II figure. Considering the Case II results, we can interpret the Case IV figure as follows. Starting from the back of the vessel, where :math:`f_a(\alpha, \theta)` is at its maximal value, :math:`f_a(\alpha, \theta)` decreases at a decreasing rate until reaching :math:`y \approx -4.6` cm. The maximal value of :math:`f_a(\alpha, \theta)` (minimal value of :math:`e(\alpha,\theta)`) being when the particle is the furthest away from the detector may be explained by the curvature of the vessel's wall. Since the wall of the vessel is curved to form a circle, the distance traveled by the photon inside the vessel on the average probable path isn't necessarily larger than the radius of the reactor. We know that at :math:`y = 0`, :math:`e(\alpha,\theta) = 10` cm. In the words, :math:`e(\alpha,\theta)` is equivalent to the radius of the reactor. On the :math:`e(\alpha,\theta)` *function of* :math:`y` figure, we can read :math:`e(\alpha,\theta) \approx 10.04` cm when :math:`y = 10` cm. We also know that an increasing distance :math:`e(\alpha,\theta)` leads to a decreasing efficiency, which means a decreasing count. Therefore, we may assume that :math:`e(\alpha,\theta)` is minimal when :math:`y \approx -10` cm or when :math:`y \approx 10` cm. And, it slowly increases until reaching :math:`y \approx -4.6` cm. When the particle reaches the :math:`y \approx -4.6` cm position (local minimum), the variation of :math:`f_a(\alpha, \theta)` is so little that :math:`f_a(\alpha, \theta)` behaves as a constant. This explains why we see the same pattern of evolution of the photon count as in Case II when :math:`y \in ]-4.6, -3.8[` cm. Similarly, when the particle sees only the face of the detector, the pattern of the counts evolution follows the same trend as the one seen on the Case II when :math:`y \in ]-3.8, 0]` cm. This also indicates very little fluctuations of :math:`e(\alpha,\theta)` as we may see on the :math:`e(\alpha,\theta)` *function of* :math:`y` figure. Therefore, the photon count is highly dependant of the weighting factors when :math:`y \in ]-3.8, 0]` cm.
```

Comment:
We notice that we have a local minimum at :math:`y \approx -4.6` where we saw the convex section on the Case II figure.

convex or inflection?

Reply:
In the words

In the other  words

Reply:
here: We also know that an increasing distance :math:`e(\alpha,\theta)` leads to a decreasing efficiency

It is good to mention that since the attention coefficient is constant everywhere inside the reactor, the efficiency would only be based on the path length, regarding the beer lambert law.